java/semantickernel-api/src/main/java/com/microsoft/semantickernel/exceptions/ConfigurationException.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.semantickernel.exceptions;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/** Kernel logic exception */
+public class ConfigurationException extends Exception {
```

Comment:
Should extend com.microsoft.semantickernel.SKException

Reply:
I would prefer this to be a checked exception...maybe we should have a CheckedSKException